autodist/kernel/graph_transformer.py

```diff
@@ -161,8 +163,10 @@ def _between_graph_apply(self, multi_gpu_graph_item: GraphItem):
             GraphItem
         """
         new_graph_item = multi_gpu_graph_item
+        new_graph_item.start_loop_optimize()
         for var_name, syncer in self._synchronizers.items():
-            new_graph_item = syncer.between_graph_apply(new_graph_item, var_name)
+            new_graph_item = syncer.between_graph_apply(new_graph_item, var_name, optimize=True)
```

Comment:
if there any way not to change the signature of `in_graph_apply()` and `between_graph_apply()` ?